include/benchmark/benchmark.h

```diff
@@ -813,6 +817,8 @@ class Benchmark {
 
   virtual void Run(State& state) = 0;
 
+  void ApplyFunctions();
```

Comment:
I'm a little surprised this is public. does it need to be?

Reply:
This method should be private.